src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let result = 0;
+
+  if (days >= 7) {
+    result = price * days;
+
+    return result - 50;
```

Comment:
Consider using a constant for the $50 discount to avoid hardcoding. This will make your code more readable and maintainable. Refer to checklist item 2.